articles/concepts/the-qubit.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: The qubit | Microsoft Docs
-description: The qubit
+title: The qubit in quantum computing
+description: Learn about qubits, the fundamental object of information in quantum computing.
```

Comment:
```suggestion
description: Learn about qubits, the fundamental unit of information in quantum computing.
```
My suggestion here is intended to avoid communicating that Q# is an OOP language. To wit, we hear people ask pretty frequently how to write "methods" in Q#.